README.md
## 派三派四 ([https://www.python1314.com/](https://www.python1314.com/))

基于Wordpress开源构建的技术博客。如需安装学习或者帮助改进，请在本地安装此项目。测试账号：admin 测试密码：0#Ix2C6MVe0Z2CZxNT。

CHANGELOG.md
## 修改记录

### 2019-11-16
- 1.优化修改Kratos主题 
- 2.新增派学工具箱和关于我们页面 

### 2019-08-10
- 1.优化修改Kratos主题 

### 2019-05-26
- 1.使用Kraos主题 
- 2.创建分类：派学日报和工具箱 
- 3.配置使用pocket和zapier实现自动化分享第三方博文到派学日报 

### 2018-10-01
- 1.使用wordpress开源系统部署在阿里云服务器


wp-config.php
<?php
/**
 * WordPress基础配置文件。
 *
 * 这个文件被安装程序用于自动生成wp-config.php配置文件，
 * 您可以不使用网站，您需要手动复制这个文件，
 * 并重命名为“wp-config.php”，然后填入相关信息。
 *
 * 本文件包含以下配置选项：
 *
 * * MySQL设置
 * * 密钥
 * * 数据库表名前缀
 * * ABSPATH
 *
 * @link https://codex.wordpress.org/zh-cn:%E7%BC%96%E8%BE%91_wp-config.php
 *
 * @package WordPress
 */

// ** MySQL 设置 - 具体信息来自您正在使用的主机 ** //
/** WordPress数据库的名称 */
define( 'DB_NAME', 'local_python1314_com' );

/** MySQL数据库用户名 */
define( 'DB_USER', 'root' );

/** MySQL数据库密码 */
define( 'DB_PASSWORD', '' );

/** MySQL主机 */
define( 'DB_HOST', 'localhost' );

/** 创建数据表时默认的文字编码 */
define( 'DB_CHARSET', 'utf8mb4' );

/** 数据库整理类型。如不确定请勿更改 */
define( 'DB_COLLATE', '' );

/**#@+
 * 身份认证密钥与盐。
 *
 * 修改为任意独一无二的字串！
 * 或者直接访问{@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org密钥生成服务}
 * 任何修改都会导致所有cookies失效，所有用户将必须重新登录。
 *
 * @since 2.6.0
 */
define( 'AUTH_KEY',         'Q!one}cs[juhvr<LJ)7A%3aC24i:nOjM%FYy5i:MnHKTINp5mT]&e8Bv(Jm)b40D' );
define( 'SECURE_AUTH_KEY',  'NUXlFdgm]R<:}3{fW{#7gBj_@B}bm#mh8idt]P{Cv+#W043F>.DVJWVv:)O0/G{2' );
define( 'LOGGED_IN_KEY',    'mV*9qO~fNjXQ#KF0yHPY#%qgKb=ze7lc|hG`two)*GG[0s*ZqSQA^9!P`Qa@Ga:7' );
define( 'NONCE_KEY',        't%7IDWoD/.zNBg][7cT4xSC:Fa:{ypG=RERo(^*<jZ65dcXE>QrZ!Khri;.}QE <' );
define( 'AUTH_SALT',        ';s4#xH<YlAZ(-y_E!&v|U1HR_AAXsj%Lm0joEX}ll`c*7mfK}I-P39&HUO4RT*AR' );
define( 'SECURE_AUTH_SALT', 'TqVLWw*`{uKFa)N+1G7!tANStto7z$IYw]0i%)q*1GQ =yDNGX9dOso0{sAw[*(m' );
define( 'LOGGED_IN_SALT',   '{WMRu3;hy,=BfCM079R1:b06wMa^~&<#s[oqW$s~wQnIY|xz49b<aMA?>DHzXo9m' );
define( 'NONCE_SALT',       '^QE[1[dt}-g,Pc8Cz C~SNwQcF+$SGkhq5*dFh<kgGIwuPv[W3?&?{Zv$Wc;ucdC' );

/**#@-*/

/**
 * WordPress数据表前缀。
 *
 * 如果您有在同一数据库内安装多个WordPress的需求，请为每个WordPress设置
 * 不同的数据表前缀。前缀名只能为数字、字母加下划线。
 */
$table_prefix = 'wp_';

/**
 * 开发者专用：WordPress调试模式。
 *
 * 将这个值改为true，WordPress将显示所有用于开发的提示。
 * 强烈建议插件开发者在开发环境中启用WP_DEBUG。
 *
 * 要获取其他能用于调试的信息，请访问Codex。
 *
 * @link https://codex.wordpress.org/Debugging_in_WordPress
 */
define('WP_DEBUG', false);

/* 好了！请不要再继续编辑。请保存本文件。使用愉快！ */

/** WordPress目录的绝对路径。 */
if ( ! defined( 'ABSPATH' ) ) {
	define( 'ABSPATH', dirname( __FILE__ ) . '/' );
}

/** 设置WordPress变量和包含文件。 */
require_once( ABSPATH . 'wp-settings.php' );


wp-content/themes/Kratos/header.php
<?php
/**
 * The template for displaying the header
 * 
 * @author Vtrois <seaton@vtrois.com>
 * @license GPL-3.0
 */
?><!DOCTYPE HTML>
<html class="no-js">
	<head>
		<title><?php wp_title( '-', true, 'right' ); ?></title>
		<meta charset="<?php bloginfo('charset'); ?>">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="robots" content="index,follow">
		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
		<meta name="format-detection" content="telphone=no, email=no">
		<meta name="description" content="<?php kratos_description(); ?>" />
		<meta name="keywords" content="<?php kratos_keywords();?>" />
		<link rel="icon" href="/favicon.ico" type="image/x-icon" />
		<meta itemprop="image" content="<?php echo kratos_thumbnail_url(); ?>"/>
		<meta name="description" itemprop="description" content="<?php kratos_description(); ?>" />
		<?php wp_head(); ?>
		<?php wp_print_scripts('jquery'); ?>
		<?php if ( kratos_option('site_bw')==1 ) : ?>
			<style type="text/css">html{filter: grayscale(100%);-webkit-filter: grayscale(100%);-moz-filter: grayscale(100%);-ms-filter: grayscale(100%);-o-filter: grayscale(100%);filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);filter: gray;-webkit-filter: grayscale(1); }
			</style>
		<?php endif; ?>
	</head>
	<?php flush(); ?>
	<body data-spy="scroll" data-target=".scrollspy">
		<div id="kratos-wrapper">
			<div id="kratos-page">
				<div id="kratos-header">
					<header id="kratos-header-section">
						<div class="container">
							<div class="nav-header">
								<?php if ( has_nav_menu('header_menu') ) :?>
									<a href="#" class="js-kratos-nav-toggle kratos-nav-toggle"><i></i></a>
								<?php endif; ?>
								<?php $site_logo = kratos_option('site_logo');?>
								<?php if ( !empty( $site_logo ) ) {?>
									<a href="<?php echo get_option('home'); ?>">
									<h1 id="kratos-logo-img"><img src="<?php echo $site_logo; ?>"></h1>
									</a>
								<?php }else{?>
									<h1 id="kratos-logo"><a href="<?php echo get_option('home'); ?>"><?php bloginfo('name'); ?></a></h1>
								<?php }?>
								<?php $defaults = array('theme_location' => 'header_menu', 'container' => 'nav', 'container_id' => 'kratos-menu-wrap', 'menu_class' => 'sf-menu', 'menu_id' => 'kratos-primary-menu', ); ?>
							 <?php wp_nav_menu($defaults); ?>
							</div>
						</div>
					</header>
				</div>

wp-content/themes/Kratos/single.php
<?php
/**
 * The template for displaying all single posts and attachments
 * 
 * @author Vtrois <seaton@vtrois.com>
 * @license GPL-3.0
 */
$sidebar = kratos_option('side_bar');
$sidebar = (empty($sidebar)) ? 'right_side' : $sidebar;
get_header();
get_header('banner'); ?>
<div id="kratos-blog-post" style="background:<?php echo kratos_option('background_index_color'); ?>">
	<div class="container">
		<div class="row">
			<?php if($sidebar == 'left_side'){ ?>
			<aside id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm scrollspy">
                <div id="sidebar">
                    <?php dynamic_sidebar('sidebar_tool'); ?>
                </div>
            </aside>
			<?php } ?>
            <section id="main" class='<?php echo ($sidebar == 'single') ? 'col-md-12' : 'col-md-8'; ?>'>
				<?php if (have_posts()) : the_post(); update_post_caches($posts); ?>
				<article>
					<div class="kratos-hentry kratos-post-inner clearfix">
						<header class="kratos-entry-header">
							<h1 class="kratos-entry-title text-center"><?php the_title(); ?></h1>
							<div class="kratos-post-meta text-center">
								<span>
								<i class="fa fa-calendar"></i> <?php echo get_the_date(); ?>
				                <i class="fa fa-eye"></i> <?php echo kratos_get_post_views();?>次阅读
				                <i class="fa fa-user-circle"></i> <?php the_author(); ?>
								</span>
							</div>
						</header>
						<div class="kratos-post-content">
						<?php if ( kratos_option('ad_show_1') ): ?>
							<a href="<?php echo kratos_option('ad_link_1'); ?>"><img src="<?php echo kratos_option('ad_img_1')?>"></a>
	                    <?php endif ?>
                        <?php the_content(); ?>
                        <div class="footer-tag clearfix">
                            <div class="pull-left">
                            <i class="fa fa-tags"></i>
                            <?php if ( get_the_tags() ) { the_tags('', ' ', ''); } else{ echo '<a>No Tag</a>';  }?>
                            </div>
                        </div>
                        <p><hr></p>
                        <div class="kratos-article-recommand">
                            <h4 class="widget-title" style="margin-bottom: 10px;">同类推荐</h4>
                            <?php
                                global $post;
                                $categories = get_the_category();
                                foreach ($categories as $category){
                            ?>
                            <ul style="padding-left:20px;">
                                <?php
                                    $posts = get_posts('numberposts=10&orderby=rand&category='. $category->term_id);
                                    foreach($posts as $post){
                                ?>
                                    <li><a href="<?php the_permalink(); ?>"><?php the_title(); ?></a></li>
                                <?php
                                    }
                                ?>
                            </ul>
                            <?php
                                }
                            ?>
                        </div>
						</div>
						<footer class="kratos-entry-footer clearfix">
							<div class="post-like-donate text-center clearfix" id="post-like-donate">
							<?php if ( kratos_option( 'post_like_donate' ) ) : ?>
				   			<a href="<?php echo kratos_option('donate_links'); ?>" class="Donate"><i class="fa fa-bitcoin"></i> 打赏</a>
				   			<?php endif; ?>
				   			<a href="javascript:;" id="btn" data-action="love" data-id="<?php the_ID(); ?>" class="Love <?php if(isset($_COOKIE['love_'.$post->ID])) echo 'done';?>" ><i class="fa fa-thumbs-o-up"></i> 点赞</a>
							<?php if ( kratos_option( 'post_share' ) ) : ?>
							<a href="javascript:;"  class="Share" ><i class="fa fa-share-alt"></i> 分享</a>
								<?php require_once( get_template_directory() . '/inc/share.php'); ?>
							<?php endif; ?>
				    		</div>
				    		<p><hr></p>
                            <?php if ( kratos_option('ad_show_2') ): ?>
                                <a href="<?php echo kratos_option('ad_link_2'); ?>"><img src="<?php echo kratos_option('ad_img_2')?>"></a>
                            <?php endif ?>
						</footer>
					</div>
					<?php if ( kratos_option( 'post_cc' )==1 ) : ?>
					<div class="kratos-hentry kratos-copyright text-center clearfix">
						<img alt="知识共享许可协议" src="<?php echo get_template_directory_uri(); ?>/images/licenses.png">
						<h5>本作品采用 <a rel="license nofollow" target="_blank" href="http://creativecommons.org/licenses/by-sa/4.0/">知识共享署名-相同方式共享 4.0 国际许可协议</a> 进行许可</h5>
					</div>
					<?php endif; ?>
					<nav class="navigation post-navigation clearfix" role="navigation">
						<?php
						$prev_post = get_previous_post(TRUE);
						if(!empty($prev_post)):?>
						<div class="nav-previous clearfix">
							<a title="<?php echo $prev_post->post_title;?>" href="<?php echo get_permalink($prev_post->ID);?>">&lt; 上一篇</a>
						</div>
						<?php endif;?>
						<?php
						$next_post = get_next_post(TRUE);
						if(!empty($next_post)):?>
						<div class="nav-next">
							<a title="<?php echo $next_post->post_title;?>" href="<?php echo get_permalink($next_post->ID);?>">下一篇 &gt;</a>
						</div>
						<?php endif;?>
					</nav>
					<?php comments_template(); ?>
				</article>
				<?php endif; ?>
			</section>
			<?php if($sidebar == 'right_side'){ ?>
				<aside id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm scrollspy">
	                <div id="sidebar">
	                    <?php dynamic_sidebar('sidebar_tool'); ?>
	                </div>
	            </aside>
			<?php } ?>
		</div>
	</div>
</div>
<?php get_footer(); ?>

wp-content/themes/Kratos/content.php
<?php
/**
 * The default template for displaying content
 * 
 * @author Vtrois <seaton@vtrois.com>
 * @license GPL-3.0
 */
$listlayout = kratos_option('list_layout');
$listlayout = (empty($listlayout)) ? 'new_layout' : $listlayout; ?>
<article class="kratos-hentry clearfix">
<?php if($listlayout == 'old_layout'){ ?>
<div class="kratos-entry-thumb">
	<?php kratos_blog_thumbnail() ?>
</div>	
<div class="kratos-post-inner">
	<header class="kratos-entry-header clearfix">
		<h2 class="kratos-entry-title"><a href="<?php the_permalink() ?>"><?php the_title(); ?></a></h2>
		<div class="kratos-post-meta">
			<span class="pull-left">
			<a href="#"><i class="fa fa-calendar"></i> <?php echo get_the_date(); ?></a>
			</span>
			<span class="visible-lg visible-md visible-sm pull-left">
			<?php $category = get_the_category(); echo '<a href="' . get_category_link($category[0] -> term_id) . '"><i class="fa fa-folder-open-o"></i> ' . $category[0] -> cat_name . '</a>'; ?>
			</span>
			<span class="pull-left">
			    <a href="<?php the_permalink() ?>"><i class="fa fa-eye"></i> <?php echo kratos_get_post_views(); ?>次阅读</a>
			</span>
		</div>
	</header>
	<div class="kratos-entry-content clearfix">
	<p><?php echo wp_trim_words(get_the_excerpt(), kratos_option('post_trim')); ?></p>
	</div>
</div>
<?php } if($listlayout == 'new_layout'){ ?>
<div class="kratos-entry-border-new clearfix">
	<div class="kratos-entry-thumb-new" style="display:none;">
		<?php kratos_blog_thumbnail_new() ?>
	</div>
	<div class="kratos-post-inner-new" style="padding-left:20px;margin-left:0;">
		<header class="kratos-entry-header-new">
			<a class="label" href="<?php $category = get_the_category();echo get_category_link($category[0] -> term_id) . '">' . $category[0] -> cat_name ; ?><i class="label-arrow"></i></a>
			<h2 class="kratos-entry-title-new"><a href="<?php the_permalink() ?>"><?php the_title(); ?></a></h2>
		</header>
		<div class="kratos-entry-content-new">
			<p><?php echo wp_trim_words(get_the_excerpt(), kratos_option('post_trim')); ?></p>
		</div>
	</div>
	<div class="kratos-post-meta-new">
		<span class="visible-lg visible-md visible-sm pull-left">
			<a href="<?php the_permalink() ?>"><i class="fa fa-calendar"></i> <?php echo get_the_date(); ?></a>
		</span>
		<span class="pull-left">
			<a href="<?php the_permalink() ?>"><i class="fa fa-eye"></i> <?php echo kratos_get_post_views(); ?>次阅读</a>
		</span>
		<span class="pull-left">
            <a href="https://www.python1314.com/author/<?php echo the_author_meta( 'user_login'); ?>" target="_blank"><i class="fa fa-user-circle"></i> <?php the_author(); ?></a>
        </span>
		<span class="pull-right">
			<a class="read-more" href="<?php the_permalink() ?>" title="查看详情">查看详情 <i class="fa fa-chevron-circle-right"></i></a>
		</span>
	</div>
</div>
<?php } ?>
</article>


wp-content/themes/Kratos/inc/share.php
<div class="share-wrap" style="display: none;">
	<div class="share-group">
		<a href="javascript:;" class="share-plain twitter" onclick="share('qq');" rel="nofollow">
			<div class="icon-wrap">
				<i class="fa fa-qq"></i>
			</div>
		</a>
		<a href="javascript:;" class="share-plain weibo" onclick="share('weibo');" rel="nofollow">
			<div class="icon-wrap">
				<i class="fa fa-weibo"></i>
			</div>
		</a>
		<a href="javascript:;" class="share-plain weixin pop style-plain" rel="nofollow">
			<div class="icon-wrap">
				<i class="fa fa-weixin"></i>
			</div>
			<div class="share-int">
				<div class="qrcode" data-url="<?php the_permalink() ?>"></div>
				<p>打开微信“扫一扫”，打开网页后点击屏幕右上角分享按钮</p>
			</div>
		</a>
	</div>
	<script type="text/javascript">
	function share(obj){
		var qqShareURL="http://connect.qq.com/widget/shareqq/index.html?";
		var weiboShareURL="http://service.weibo.com/share/share.php?";
		var facebookShareURL="https://www.facebook.com/sharer/sharer.php?";
		var twitterShareURL="https://twitter.com/intent/tweet?";
		var googleplusShareURL="https://plus.google.com/share?";
		var host_url="<?php the_permalink(); ?>";
		var title="【<?php the_title(); ?>】";
		var qqtitle="<?php the_title(); ?>";
		var excerpt="<?php echo get_the_excerpt(); ?>";
		var pic="<?php echo share_post_image(); ?>";
		var _URL;
		if(obj=="qq"){
			_URL=qqShareURL+"url="+host_url+"&title="+qqtitle+"&pics="+pic+"&desc=&summary="+excerpt+"&site=vtrois";
		}else if(obj=="weibo"){
			_URL=weiboShareURL+"url="+host_url+"&title="+title+excerpt+"&pic="+pic;
		}else if(obj=="facebook"){
	 		_URL=facebookShareURL+"u="+host_url;
		}else if(obj=="twitter"){
	 		_URL=twitterShareURL+"text="+title+excerpt+"&url="+host_url;
		}else if(obj=="googleplus"){
	 		_URL=googleplusShareURL+"url="+host_url;
		}
		window.open(_URL);
	}
	</script>
</div>

wp-content/themes/Kratos/page-daily.php
<?php
/*
Template Name: 日报模版
*/
$page_side_bar = kratos_option('page_side_bar');
$page_side_bar = (empty($page_side_bar)) ? 'right_side' : $page_side_bar;
get_header();
get_header('banner'); ?>
<div id="kratos-blog-post" style="background:<?php echo kratos_option('background_index_color'); ?>">
	<div class="container">
		<div class="row">
			<?php if($page_side_bar == 'left_side'){ ?>
				<aside id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm scrollspy">
	                <div id="sidebar">
	                    <?php dynamic_sidebar('sidebar_tool'); ?>
	                </div>
	            </aside>
			<?php } ?>
            <section id="main" class='<?php echo ($page_side_bar == 'single') ? 'col-md-12' : 'col-md-8'; ?>'>
				<article>
					<div class="kratos-hentry kratos-post-inner clearfix">
          <header class="kratos-entry-header">
            <h1 class="kratos-entry-title text-center">派学日报 第<?php echo floor((strtotime(date("Y-m-d"))-strtotime("2018-10-01"))/86400) ?>期</h1>
            <div class="kratos-post-meta text-center">
              <span>
                <i class="fa fa-calendar"></i> <?php echo date("Y年m月d日"); ?>
                <i class="fa fa-eye"></i> <?php echo kratos_get_post_views();?>次浏览
              </span>
            </div>
          </header>
            <div class="kratos-post-content">
              <p>每天，我们会从大量信息源中为你精心挑选有价值阅读的技术文章，中英文都会有 – 都是有关编程总结、技术趋势、开发技巧等主题的文章。

              <p>当然，还会有很多有趣的东西，大家拭目以待吧:)</p>

              <p><hr></p>
              <?php if ( kratos_option('ad_show_1') ): ?>
                <a href="<?php echo kratos_option('ad_link_1'); ?>"><img src="<?php echo kratos_option('ad_img_1')?>"></a>
              <?php endif ?>
              <?php
                $args=array(
                    'cat' => 37,   // 分类ID
                    'posts_per_page' => 10, // 显示篇数
                    'post_date' => date("Y-m-d")
                );
                query_posts($args);
                if(have_posts()) : while (have_posts()) : the_post();
              ?>
                <p style="margin-bottom:5px"><b><?php the_title(); ?></b></p>
                <p style="margin-bottom:5px"><?php if ( get_the_tags() ) { the_tags('标签：', ' ', ''); }?></p>
                <p style="margin-bottom:25px">简介：<?php echo wp_trim_words(get_the_excerpt(), 68); ?><a target="_blank" href="<?php $reg="/href=\"([^\"]+)/";preg_match($reg, get_the_content(), $a); echo substr($a[0], 6); ?>">原文链接</a></p>
              <?php  endwhile; endif; wp_reset_query(); ?>
              <p class="text-center"><a class="more" target="_blank" href="/learn/daily" style="border: 1px solid #ef7042; padding: 4px 12px;"><i class="fa fa-random" style="font-size: 12px; margin-right: 4px;"></i>往期更多</a></p>
              <p><hr></p>
              <?php if ( kratos_option('ad_show_2') ): ?>
                <a href="<?php echo kratos_option('ad_link_2'); ?>"><img src="<?php echo kratos_option('ad_img_2')?>"></a>
              <?php endif ?>
						</div>
						<?php if(kratos_option('page_like_donate')||kratos_option('page_share')) {?>
						<footer class="kratos-entry-footer clearfix">
								<div class="post-like-donate text-center clearfix" id="post-like-donate">
								<?php if ( kratos_option( 'page_like_donate' ) ) : ?>
					   			<a href="<?php echo kratos_option('donate_links'); ?>" class="Donate"><i class="fa fa-bitcoin"></i> 打赏</a>
					   			<?php endif; ?>
								<?php if ( kratos_option( 'page_share' ) ) : ?>
								<a href="javascript:;"  class="Share" ><i class="fa fa-share-alt"></i> 分享</a>
									<?php require_once( get_template_directory() . '/inc/share.php'); ?>
								<?php endif; ?>
					    		</div>
						</footer>
						<?php }?>
					</div>
						<?php if ( kratos_option( 'page_cc' ) ) : ?>
						<div class="kratos-hentry kratos-copyright text-center clearfix">
							<img alt="知识共享许可协议" src="<?php echo get_template_directory_uri(); ?>/images/licenses.png">
							<h5>本作品采用 <a rel="license nofollow" target="_blank" href="http://creativecommons.org/licenses/by-sa/4.0/">知识共享署名-相同方式共享 4.0 国际许可协议</a> 进行许可</h5>
						</div>
						<?php endif; ?>
						<?php comments_template(); ?>
				</article>
			</section>
			<?php if($page_side_bar == 'right_side'){ ?>
			<aside id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm scrollspy">
                <div id="sidebar">
                    <?php dynamic_sidebar('sidebar_tool'); ?>
                </div>
            </aside>
			<?php } ?>
		</div>
	</div>
</div>
<?php get_footer(); ?>

wp-content/themes/Kratos/page-toolbox.php
<?php
/*
Template Name: 工具箱模版
*/
get_header();
get_header('banner'); ?>
<div id="kratos-blog-toolbox" style="background:<?php echo kratos_option('background_index_color'); ?>">
	<div class="container">
        <aside id="kratos-widget-area" class="col-md-4 hidden-xs hidden-sm scrollspy">
            <div id="sidebar"  class="col-md-12 module module-blog">
                <h4>派学日记 <a class="more" target="_blank" title="阅读更多" href="/learn/daily"><i class="fa fa-angle-right"></i></a></h4>
                <ul>
                <?php
                    $args=array(
                        'cat' => 37,   // 分类ID
                        'posts_per_page' => 15, // 显示篇数
                    );
                    query_posts($args);
                    if(have_posts()) : while (have_posts()) : the_post();
                ?>
                    <li>
                        <a target="_blank" href="<?php $reg="/href=\"([^\"]+)/";preg_match($reg, get_the_content(), $a); echo substr($a[0], 6); ?>">
                            <?php echo wp_trim_words(the_title(), 16); ?>
                        </a>
                    </li>
                <?php  endwhile; endif; wp_reset_query(); ?>
                </ul>
            </div>
        </aside>
        <section id="main" class='col-md-8'>
            <div class="row module module-tool" style="margin-bottom: 15px;">
                <div class="col-md-12">
                    <h4>派学利器 <select class="more"><option>博客导航</option><option>工具链接</option><option>书籍推荐</option></select></h4>
                    <ul>
                    <?php
                        $args=array(
                            'cat' => 36,   // 分类ID
                            'posts_per_page' => 6, // 显示数量
                        );
                        query_posts($args);
                        if(have_posts()) : while (have_posts()) : the_post();
                    ?>
                        <li class="col-md-2 col-xs-6">
                            <a target="_blank" class="link" href="<?php echo wp_trim_words(get_the_excerpt(), kratos_option('post_trim')); ?>">
                                <?php the_post_thumbnail(); ?>
                                <span><?php the_title(); ?></span>
                            </a>
                        </li>
                    <?php  endwhile; endif; wp_reset_query(); ?>
                    </ul>
                </div>
            </div>
            <div class="row module">
                <div class="col-md-12">
                    <h4>Github精选 <select class="more"><option>All</option><option>JavaScript</option><option>Python</option><option>Java</option></select></h4>
                    <ul style="padding-bottom: 15px;">
                        <li class="col-md-12 text-center">加载失败</li>
                    </ul>
                </div>
            </div>
        </section>
	</div>
</div>
<?php get_footer(); ?>